src/content/docs/fr/guides/images.mdx

```diff
@@ -475,6 +475,25 @@ export default defineConfig({
 });
 ```
 
+## Mise en cache des assets
```

Comment:
In the explanation you have translated `assets` with `ressources`, so maybe we should also translate it here:
```suggestion
## Mise en cache des ressources
```